style.css

```diff
@@ -1,114 +1,215 @@
+.row{
+    display: flex;
+    min-height: 100vh;
+    overflow: hidden;
+}
+
 .left,.right{
-    height: 50vh ;
+    align-items:center;
+    height: 100% ;
+    justify-content:center;
     width: 100%;
 }
 
-@media screen and (min-width:768px){
-    .left,.right{
+.form-2,.l-footer{
+    display: none;
+}
+
+@media only screen and (min-width:766px) and (max-width:991px){
+    .on{
+        color: azure;
+        font-size: 60px;
+        position: absolute;
+    }
+    .dls{
+        font-size: 35px;
+    }
+    .app{
+        font-size: 40px;
+        margin-top: 10px;
+    }
+    .col-lg-8{
+        display: none;
+    }
+    
+    .left{
+        display: flex;
         height: 100vh;
     }
+
+    .form-2{
+        background-color: white;
+        border-radius: 20px;
+        display: flex;
+        flex-direction: column;
+        height: 40vh;
+        justify-content: center;
+        margin-top: 380px;
+        width: 50%;
+    }
+    
+    h4{
+        display: flex;
+        font-size: 40px;
+        height: 20%;
+        justify-content: center;
+        width: 100%;
+    }
+    
+    .left form{
+        display: flex;
+        flex-direction: column;
+        height: 60%;
+        justify-content: space-around;
+        padding-right: 50px;
+        padding-left: 50px;
+        width: 100%;
+    }
+
+    .para{
+        color: azure;
+        display: flex;
+        font-size: 27px;
+        font-family: 'Roboto', sans-serif;
+        margin-bottom: 270px;
+        position: absolute;
+        text-align: center;
+    }
+
+    .l-footer{
+        bottom: 90px;
+        color: white;
+        display: flex;
+        font-size: 14px;
+        justify-content: center;
+        position: absolute;
+        text-align: center;
+        width: 100%;
+    }
 }
 
 .left{
     background-color: blueviolet;
-    
 }
 
 .right{
     background-color: white;
     flex-direction: column;
+    
 }
-.left h2{
+
+.dls{
+    color: azure;
     position: absolute;
     top:60px;
-    left:60px;
-    color: azure;
 }
-.left h1{
+
+.on{
+    /* border-bottom: 1px solid white; */
```

Comment:
If this style is not needed then please remove it instead of commenting. Comments are used for helping other developers to understand the code. So we should not leave the code commented in the application.